systems/WikibaseClient/05-Building_Block_View.md

```diff
@@ -3,3 +3,61 @@
 ## Whitebox Overall System
 
 ![Alt Text](./diagrams/05-building-blocks.drawio.svg)
+
+## User Interfaces
+
+## APIs
+
+### Action API Description
+
+![Alt Text](./diagrams/05-api-description.drawio.svg)
```

Comment:
Instead of just saying "Alt Text" which looks a bit silly it would be nice if they actually contained a very short description.